wagtail/admin/templates/wagtailadmin/pages/_unsaved_changes_warning.html

```diff
@@ -0,0 +1,12 @@
+{% load wagtailadmin_tags i18n %}
+<li class="footer__container footer__container--hidden footer__save-warning" data-unsaved-warning>
+    <div hidden data-unsaved-type="any" class="footer__save-warning">
+        {% icon name="warning" class_name="default footer__emphasis" %}
+        <div>
+            <b hidden data-unsaved-type="edits">{% blocktrans %}You have <span class="footer__emphasis">unsaved edits</span>{% endblocktrans %}</b>
+            <b hidden data-unsaved-type="comments">{% blocktrans %}You have <span class="footer__emphasis">unsaved comments</span>{% endblocktrans %}</b>
+            <b hidden data-unsaved-type="all">{% blocktrans %}You have <span class="footer__emphasis">unsaved edits</span> and <span class="footer__emphasis">comments</span>{% endblocktrans %}</b>
+            <p>{% trans 'Save the page before leaving' %}</p>
+        </div>
+    </div>
+</li>
```

Comment:
Missing a newline here, could you configure your editor to add these automatically?